buildSrc/src/main/kotlin/com/datadog/gradle/config/AndroidConfig.kt

```diff
@@ -19,7 +19,7 @@ object AndroidConfig {
     const val MIN_SDK_FOR_WEAR = 23
     const val BUILD_TOOLS_VERSION = "33.0.0"
 
-    val VERSION = Version(1, 19, 1, Version.Type.Release)
+    val VERSION = Version(1, 19, 2, Version.Type.Release)
```

Comment:
don't you want to add this in another PR ?

Reply:
no, it is in the separate commit, so this is enough